editor.planx.uk/src/@planx/components/MapAndLabel/Editor.tsx

```diff
@@ -0,0 +1,28 @@
+import { ComponentType as TYPES } from "@opensystemslab/planx-core/types";
+import { EditorProps } from "@planx/components/ui";
+import { useFormik } from "formik";
+import React from "react";
+
+import { MapAndLabel, parseContent } from "./model";
+
+type Props = EditorProps<TYPES.MapAndLabel, MapAndLabel>;
+
+export default MapAndLabelComponent;
+
+function MapAndLabelComponent(props: Props) {
+  const formik = useFormik({
+    initialValues: parseContent(props.node?.data),
+    onSubmit: (newValues) => {
+      props.handleSubmit?.({
+        type: TYPES.MapAndLabel,
+        data: newValues,
+      });
+    },
+  });
+
+  return (
+    <form onSubmit={formik.handleSubmit} id="modal">
+      //...
```

Comment:
nit: I'd expect to see a little bit more initially outlined here - see below comment re: title & description. Default props also already include internal notes! 

Maybe more like this:
```
    <form onSubmit={formik.handleSubmit} id="modal">
      <ModalSection>
        <ModalSectionContent
          title="Map and label"
          Icon={ICONS[TYPES.MapAndLabel]}
        >
          <InputGroup>
            <InputRow>
              <Input
                format="large"
                name="title"
                placeholder={formik.values.title}
                value={formik.values.title}
                onChange={formik.handleChange}
              />
            </InputRow>
            <InputRow>
              <RichTextInput
                name="description"
                placeholder="Description"
                value={formik.values.description}
                onChange={formik.handleChange}
              />
            </InputRow>
            <InputRow>
              <Input
                format="data"
                name="fn"
                placeholder={formik.values.fn}
                value={formik.values.fn}
                disabled
              />
            </InputRow>
          </InputGroup>
        </ModalSectionContent>
      </ModalSection>
      <InternalNotes
        name="notes"
        value={formik.values.notes}
        onChange={formik.handleChange}
      />
    </form>
```